scripts/upgrade/upgrade-facets.ts

```diff
@@ -570,4 +638,17 @@ async function executeFacetUpdates(
   } else {
     console.log("ℹ️  No facet cuts to execute");
   }
+
+  // Update MetaTransactionFacet allowlist
+  if (config.metaTxAllowlist) {
+    const metaTransactionFacet = await hre.ethers.getContractAt("MetaTransactionFacet", protocolAddress);
+
+    if (config.metaTxAllowlist.add && config.metaTxAllowlist.add.length > 0) {
```

Comment:
I think we should first do `remove` and then `add` case. 

For example, we have `contract A` with function `foo()`. But then A becomes too big and we split it into two contracts `contract A` and `contract B` and function `foo()` gets moved to `contract B`.  

Since it was removed from A and added to B, it will be on both `add` and `remove` lists. But if the remove is done 2nd, in the end state `foo()` won't be allowlisted even though it should be.

Alternatively, you can do one last bit of preprocessing, and remove all duplicates from `config.metaTxAllowlist.add` and `config.metaTxAllowlist.remove.length` and then the orderd does not matter.